### PR TITLE
`tailor-gui`: temporary fix for UI bugs

### PR DESCRIPTION
**Pull request by SimonBrandner**
## Description of changes

Temporary fix for https://github.com/AaronErhardt/tuxedo-rs/issues/46

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/os-specific/linux/tailor-gui/default.nix [hunk 1]
```diff
@@ -46,6 +46,11 @@ stdenv.mkDerivation {
     gtk4
   ];
 
+  postFixup = ''
+    wrapProgram $out/bin/tailor_gui \
+      --set GDK_BACKEND "x11"
+  '';
```
Comment (mrcjkb) [inline pkgs/os-specific/linux/tailor-gui/default.nix:52]: suggestion: leave a comment linking to https://github.com/AaronErhardt/tuxedo-rs/issues/46,
with a `TODO` to remove this when it has been fixed.

**Review by mrcjkb — CHANGES_REQUESTED** [review]
Thanks for the PR.

LGTM. I can't test this on Wayland, but it still builds an runs fine on X11

Just a small request  :smile:

**Comment by mrcjkb** [comment]
Result of `nixpkgs-review pr 277548` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>tailor-gui</li>
  </ul>
</details>

**Comment by SimonBrandner** [comment]
Superseded by https://github.com/AaronErhardt/tuxedo-rs/pull/60

**Pull request closed without merging**